roadmap.md

```diff
@@ -5,56 +5,10 @@ title: Roadmap
 
 # Bazel Roadmap
 
-## Bazel Core Improvement Areas 
-
-As of July 2018, critical areas that Bazel team addresses within Bazel core to
-fulfill the [Bazel Vision](https://docs.bazel.build/versions/master/bazel-vision.html)
-are as follows:
-
-*   **Remote execution API** is under development, but we are not yet ready to
-    provide the full guidance to rule authors on how to write rules to enable it
-    for their language.
-*   [**External dependency management**](roadmaps/external-deps.html) story 
-    needs to be improved including both
-    the **general dependency management** mechanism in Bazel and **interfacing with
-    external package managers and build systems**. The latter is critical for
-    _gradual adoption_ of Bazel.
-*   [**Configurability**](roadmaps/configuration.html). Outside of a few
-    hardcoded cases (fat apk for Android,host -> target transitions), the promise
-    of single-command multi-platform builds is unfulfilled. 
-*   [**Starlark API and tooling**](roadmaps/starlark.html). We need to support the rule authors with
-    **clear, documented, stable API**, and **strong language and tooling**:
-    documentation generators, refactoring tools, testing frameworks and so on.
-
 ## Roadmaps for specific Bazel areas
 *  [Android](https://github.com/bazelbuild/rules_android/blob/master/ROADMAP.md)
```

Comment:
Android is also outdated.

Reply:
@timpeut - any update on this?

(I skipped the annotation here because it's separate repository.)